source/slang/slang-serialize-container.cpp

```diff
@@ -348,6 +348,12 @@ namespace Slang {
             }
         }
 
+	// TODO:
```

Comment:
It doesn't look like this is causing any CI test failures, but I've put the "breaking change" label on for now because of the hint that I got that there might be users that are somehow relying on the program level IR in slang-modules. Not sure what to do about this.

Reply:
I don't know how the program IR can be used to even broach the subject.  What would OV be doing to depend on this feature?

I think it might be a viable option to prevent the generation of the program level IR in the first place when embedding precompiled binaries into modules.  Basically, don't call generateOutput() when -embed-dxil/spirv is specified, but I'm unsure.

Reply:
@csyonghe Any feedback on this change to cut out the "target component" IR from slang-modules?

Reply:
I actually don't know. I guess we really need to run this against ov tests to find out whether or not this is breaking OV.

Reply:
It surprises me to find that my change does indeed cause OV regressions, but not because of this.  Taking my change and turning this code back on doesn't help my test results, and taking TOT and turning this code off doesn't hurt test results.  Something else in the change is causing OV regressions, and I'm looking into it now.

Reply:
Resolved, this doesn't break known usecases with OV.